src/models/entity-contact.js

```diff
@@ -48,6 +48,14 @@ const entityContact = (sequelize, DataTypes) => {
         }
     }
 
+    EntityContact.findByEntityId = async (entityId) => {
+        const entries = await EntityContact.findAll({
+            where: {entityId}
```

Comment:
Does this automatically find it by pk?

Reply:
It does not for some reason. Not sure what was off about that but it wasn't really worth debugging. When I used `findByPk` nothing came back.